src/mount/chunk_writer.h

```diff
@@ -175,6 +180,7 @@ class ChunkWriter {
 	bool acceptsNewOperations_;
 	int combinedStripeSize_;
 	int dataChainFd_;
+	int chunkSizeInBlocks_;
```

Comment:
Consider declaring this variable as unsigned interger as negative values have not practical meaning for this variable.

Reply:
Same here.